Src/sounds.c

```diff
@@ -97,7 +97,12 @@ void playStartupTune()
     __disable_irq();
 
     uint8_t value = *(uint8_t*)(EEPROM_START_ADD + 48);
-    if (value != 0xFF) {
+#ifdef WCH
```

Comment:
why is this change needed?

Reply:
Because after it is  erased, its default value is 0xe339e339

Reply:
ok, better to have a define ERASED_FLASH_BYTE, we could default it to 0xff at the top of sounds.c, but use ifndef to allow a Mcu port to override it

Reply:
yeah  would be better